packages/features/ee/teams/lib/payments.ts

```diff
@@ -121,15 +124,23 @@ export const purchaseTeamOrOrgSubscription = async (input: {
   let priceId: string | undefined;
 
   if (pricePerSeat) {
-    const customPriceObj = await getPriceObject(fixedPrice);
-    priceId = await createPrice({
-      isOrg: !!isOrg,
-      teamId,
-      pricePerSeat,
-      billingPeriod,
-      product: customPriceObj.product as string, // We don't expand the object from stripe so just use the product as ID
-      currency: customPriceObj.currency,
-    });
+    if (
+      isOrg &&
+      pricePerSeat === DEFAULT_SELF_SERVE_ORG_PRICE_PER_SEAT &&
+      seats < DEFAULT_SELF_SERVE_ORG_SEATS
+    ) {
+      priceId = fixedPrice as string;
+    } else {
+      const customPriceObj = await getPriceObject(fixedPrice);
+      priceId = await createPrice({
+        isOrg: !!isOrg,
+        teamId,
+        pricePerSeat,
+        billingPeriod,
+        product: customPriceObj.product as string, // We don't expand the object from stripe so just use the product as ID
+        currency: customPriceObj.currency,
+      });
+    }
```

Comment:
Do we not want to avoid it for custom prices that end up being same?

Reply:
No - not yet. I talked about this with shiraz/peer and we agreed that any users that are on a custom plan for now we want to leave it as a custom price incase we need to adjust it for them manually in the future e.g. make a new deal for them etc